bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -22,6 +22,14 @@ const VideoListItem = (props) => {
   const [videoIsReady, setVideoIsReady] = useState(false);
   const [isStreamHealthy, setIsStreamHealthy] = useState(false);
   const [isMirrored, setIsMirrored] = useState(false);
+  const [isVideoSqueezed, setIsVideoSqueezed] = useState(false);
+
+  const resizeObserver = new ResizeObserver((entry) => {
```

Comment:
I see you call this via `resizeObserver.observe(videoContainer.current);` so I wonder if there'd be a case where we have undefined `entry` or `entry[0]` or `entry[0].contentRect`?